src/runtime-rs/crates/service/src/manager.rs

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019-2022 Alibaba Cloud
+// Copyright (c) 2019-2022 Ant Group
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::{
+    os::unix::io::{FromRawFd, RawFd},
+    sync::Arc,
+};
+
+use anyhow::{Context, Result};
+use common::message::{Action, Message};
+use containerd_shim_protos::shim_async;
+use runtimes::RuntimeHandlerManager;
+use tokio::sync::mpsc::{channel, Receiver};
+use ttrpc::asynchronous::Server;
+
+use crate::task_service::TaskService;
+
+/// message buffer size
+/// considering that there will not be too many server messages, currently use 8
```

Comment:
```suggestion
/// considering that there will not be too many server messages.
```

Also, the comment makes me wonder if this should be a config option?

Reply:
Thanks for your review, and updated the comment.